### PR TITLE
chore: update error-extender lib

### PR DESCRIPTION
**Pull request by stas-nc**

### CODE REVIEW

File: ilc/server/tailor/fragment-hooks.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ function insertStart(logger, stream, attributes, headers) {
                     isAsync
                         ? `<!-- Async fragments are not fully implemented yet: ${uri} -->`
                         : id
-                        ? asyncStylesLoadTemplate(uri, id)
-                        : '',
+                          ? asyncStylesLoadTemplate(uri, id)
+                          : '',
```
Comment (harnyk) [inline ilc/server/tailor/fragment-hooks.js:50]: I think our auto-formatting tools differ. I use Prettier, and it does not add the indents. Which tool is considered a standard in this codebase?

File: registry/server/settings/services/SettingsService.ts [hunk 2]
```diff
@@ -122,15 +122,9 @@ export class SettingsService {
...
-                'settings.default',
-                'settings.scope',
-                'settings.secret',
-                'settings.meta',
-                'settings_domain_value.value as value',
-                'settings_domain_value.domainId as domainId',
-            )
+            .select<
+                SettingRaw[]
+            >('settings.key', 'settings.default', 'settings.scope', 'settings.secret', 'settings.meta', 'settings_domain_value.value as value', 'settings_domain_value.domainId as domainId')
```
Comment (harnyk) [inline registry/server/settings/services/SettingsService.ts:127]: the same formatting issue as above

File: ilc/server/tailor/fragment-hooks.js (hunk 1 above)
Comment (stas-nc) [reply ilc/server/tailor/fragment-hooks.js:50]: In this repo prettier is used. But please check if you have the latest node_modules, there was a recent update there that affected formatting.

Comment (harnyk) [reply ilc/server/tailor/fragment-hooks.js:50]: indeed

**Review by yehor-manzhula — APPROVED** [review]

File: ilc/common/utils.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+const errorExtender = require('@namecheap/error-extender');
```
Comment (harnyk) [inline ilc/common/utils.js:1]: why not import?

Comment (stas-nc) [reply ilc/common/utils.js]: switched to import and ts

**Review by harnyk — APPROVED** [review]

**Pull request merged**